dev/tools/localization/bin/gen_localizations.dart

```diff
@@ -2,11 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// This program generates a getMaterialTranslation() and a
-// getCupertinoTranslation() function that look up the translations provided by
+// This program generates getMaterialTranslation(), getCupertinoTranslation(),
+// and getWidgetsTranslation functions that look up the translations provided by
```

Comment:
nit: stick to one style, either use `getFooTranslation` or `getFooTranslation()` everywhere in this enumeration?

Reply:
added ()